hack-sound-server/data/com.endlessm.HackSoundServer2.xml

```diff
@@ -0,0 +1,28 @@
+<node>
```

Comment:
You don't need this file in the `hack-sound-server` directory (it's the iface for the v2)